website/docs/configuration/observability.md

```diff
@@ -21,9 +21,12 @@ FerretDB provides the following log formats:
 <!-- https://github.com/FerretDB/FerretDB/issues/4438 -->
 
 - `console` is a human-readable format with optional colors;
+- `mongo` is machine-readable structured JSON format, similar to the one used in MongoDB.
+  It follows the Relaxed Extended JSON specification.
+  Fields required in the output format but not yet implemented, will be included empty, to maintain compatibility;
 - `text` is machine-readable [logfmt](https://brandur.org/logfmt)-like format
   (powered by [Go's `slog.TextHandler`](https://pkg.go.dev/log/slog#TextHandler));
-- `json` if machine-readable JSON format
+- `json` is machine-readable JSON format
   (powered by [Go's `slog.JSONHandler`](https://pkg.go.dev/log/slog#JSONHandler)).
```

Comment:
Is that the best order?
When should semicolons be used in a list?

Reply:
> When should semicolons be used in a list?

When items could be concatenated to form a single sentence:

- John, the baker;
- Simon, the policeman;
- Toby, the architect;

=>

John, the baker; Simon, the policeman; and Toby, the architect.

Using semicolons makes it weird if a list item contains multiple complete sentences. Just use periods.